### ISSUE TITLE
Sidenav like vue-material site

### ISSUE THREAD

**Issue by Dennisoost** [marcosmoura/vue-material#44]
Hi!
So i was wondering if there is a (built-in) way to show the side nav like on the vue-material site?
https://marcosmoura.github.io/vue-material/#/ 
Can't seem to find it myself.

**Comment by marcosmoura** [comment]
Sorry, but I didn't understand. You mean the expansion lists? The sidenav sticked on the desktop version? Or the header with the logo?

**Comment by mixedCase** [comment]
@marcosmoura In simpler terms, is the source code for the website uploaded anywhere to see?

**Comment by marcosmoura** [comment]
@mixedCase Yes. Of course. It's on the ```docs``` folder of this repository. :)

**Comment by marcosmoura** [comment]
Well. Right now Vue Material doesn't have any way (built in) to make the ```md-sidenav``` opened on desktop by default. I have plans to add this on the future along with other sidenav examples.
You can have the same thing adding a media query to your own css:
``` sass
@media (min-width: 1280px) {
  .your-page-container {
    padding-left: 304px;
  }
}
```

The idea is to push the content of your page creating a space to "hold" the sidenav.
The trick is here: https://github.com/marcosmoura/vue-material/blob/master/docs/src/App.vue#L175

Let me know if you have success on this.
Put your code on codepen and I will fix this for you! :D
Thanks!

**Comment by mixedCase** [comment]
Thanks, but unfortunately no success. Nothing shows up in the space left by the padding.

Regardless, I simply reverted to a more performant solution I was previously using and did away with `md-sidenav`. Hope to see that functionality land! :)

**Comment by marcosmoura** [comment]
Oh. I think I forgot another thing:
``` sass
@media (min-width: 1280px) {
  .md-sidenav-content {
    top: 0;
    pointer-events: auto;
    transform: translate3d(0, 0, 0); //This is the trick!
    box-shadow: $material-shadow-2dp;
  }
}
```

Try this for one more time, please.

**Comment by mixedCase** [comment]
I have no way of importing the $material-shadow-2dp variable that I can think of.

**Comment by marcosmoura** [comment]
Yep. You're right. You don't even need this shadow. :)

**Comment by mixedCase** [comment]
I'm afraid I already rearchitected everything to use flexbox with the old solution and honestly too tired to retry making sure that my own css doesn't prevent md-sidebar from working. I apologize :( But thanks for your help!

**Comment by marcosmoura** [comment]
Uh. Too bad! :(
I hope that you can try this later. I will try to prioritize this md-sidenav change.

**Comment by Morgul** [comment]
Just to chime in, the `transform: translate3d(0, 0, 0);` does the trick perfectly. For now, it's a reasonable work-around.

**Comment by kayyyy** [comment]
+1 to have such a sidebar.

**Comment by marcosmoura** [comment]
#154 #275

**Comment by Eder87rh** [comment]
@marcosmoura @mixedCase 

transform: translate3d(0, 0, 0) !important; 

works for me only qith the !important css text :) thanks

this is my component
`<template>
    <div class="container">
        <md-toolbar>
            <md-button class="md-icon-button" @click="toggleMainSidebar">
                <md-icon>menu</md-icon>
            </md-button>

            <h2 class="md-title" style="flex: 1">CRM</h2>

            <md-button class="md-button">
                Cerrar sesión
            </md-button>
        </md-toolbar>

        <md-sidenav class="main-sidebar md-left md-fixed" md-swipeable ref="main-sidebar" @open="open('Left')" @close="close('Left')">
            <md-toolbar >
                <div class="md-toolbar-container">
                    <h2 class="md-title">CRM</h2>
                </div>
            </md-toolbar>

            <div class="main-sidebar-links">
                <md-list class="md-dense">
                    <md-list-item>
                        <router-link exact to="/">Introduction</router-link>
                    </md-list-item>

                    <md-list-item>
                        <router-link exact to="/getting-started">Getting Started</router-link>
                    </md-list-item>

                    <md-list-item>
                        <span>Themes</span>

                        <md-list-expand>
                            <md-list>
                                <md-list-item class="md-inset">
                                    <router-link exact to="/themes/configuration">Configuration</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/themes/dynamic-themes">Dynamic Theme</router-link>
                                </md-list-item>
                            </md-list>
                        </md-list-expand>
                    </md-list-item>

                    <md-list-item>
                        <span>Components</span>

                        <md-list-expand>
                            <md-list>
                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/avatar">Avatar</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/bottom-bar">Bottom Bar</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/button">Button</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/button-toggle">Button Toggle</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/card">Card</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/checkbox">Checkbox</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/chips">Chips</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/dialog">Dialog</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/file">File</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/icon">Icon</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/image-loader">Image Loader</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/ink-ripple">Ink Ripple</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/input">Input</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/list">List</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/menu">Menu</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/onboarding">Onboarding</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/progress">Progress</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/radio">Radio</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/rating-bar">Rating Bar</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/select">Select</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/sidenav">Sidenav</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/snackbar">Snackbar</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/speed-dial">Speed Dial</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/spinner">Spinner</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/stepper">Stepper</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/subheader">Subheader</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/switch">Switch</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/tabs">Tabs</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/table">Table</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/toolbar">Toolbar</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/tooltip">Tooltip</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/components/whiteframe">Whiteframe</router-link>
                                </md-list-item>
                            </md-list>
                        </md-list-expand>
                    </md-list-item>

                    <md-list-item>
                        <span>UI Elements</span>
                        <md-list-expand>
                            <md-list>
                                <md-list-item class="md-inset">
                                    <router-link exact to="/ui-elements/layout">Layout</router-link>
                                </md-list-item>

                                <md-list-item class="md-inset">
                                    <router-link exact to="/ui-elements/typography">Typography</router-link>
                                </md-list-item>
                            </md-list>
                        </md-list-expand>
                    </md-list-item>

                    <md-list-item>
                        <router-link exact to="/changelog">Changelog</router-link>
                    </md-list-item>

                    <md-list-item>
                        <router-link exact to="/about">About</router-link>
                    </md-list-item>
                </md-list>
            </div>
        </md-sidenav>

        <main class="main-content">

                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quo, rerum? Error sunt, aperiam dolores, atque
                expedita molestiae tenetur. Quis eveniet accusamus velit explicabo adipisci reiciendis modi eaque quas,
                officia excepturi.</p>

        </main>
    </div>
</template>

<script>
    import VueMaterial from 'vue-material'

    export default {
        data(){
            return {}
        },
        components:{
            VueMaterial
        },
        methods: {
            toggleMainSidebar() {
                this.$refs['main-sidebar'].toggle();
            },
            toggleRightSidenav() {
                this.$refs.rightSidenav.toggle();
            },
            closeRightSidenav() {
                this.$refs.rightSidenav.close();
            },
            open(ref) {
                console.log('Opened: ' + ref);
            },
            close(ref) {
                console.log('Closed: ' + ref);
            }
        }
    }
</script>

<style>
    @media (min-width: 1280px) {
        .main-content {
            padding-left: 304px;
        }
    }

    @media (min-width: 1280px) {
        .md-sidenav-content {
            top: 0;
            pointer-events: auto;
            transform: translate3d(0, 0, 0) !important;
        }
    }
</style>`

**Comment by miclefebvre** [comment]
Needed the !important too. Thanks
